### PR TITLE
Fixes #2586 NTR-enteroendocrine progenitor

### LINKED ISSUE #2586: [NTR] intestinal enteroendocrine progenitor
Please check that the term does not already exist by using the ontology search tool OLS:
https://www.ebi.ac.uk/ols4/ontologies/cl

**Preferred term label**
intestinal enteroendocrine progenitor 

**Synonyms** (add reference(s), please)
EEC Progenitor (related synonym) PMID: [31756561](https://pubmed.ncbi.nlm.nih.gov/31756561)

**Definition** (free text, with reference(s), please. PubMed ID format is PMID:XXXXXX)

An epithelial cell that is part of the crypt of Lieberkuhn, originating from intestinal stem cells and giving rise to enteroendocrine cells (EECs). It expresses markers like Neurog3, Fev, and Prox1, and has the ability to proliferate and differentiate into various EEC subtypes. Its proliferative potential supports both crypt growth and cell proliferation, distinguishing it from fully differentiated EECs.

PMID: [31756561](https://pubmed.ncbi.nlm.nih.gov/31756561)
PMID: [32531023](https://pubmed.ncbi.nlm.nih.gov/32531023)
PMID: [35913117](https://pubmed.ncbi.nlm.nih.gov/35913117)
doi:/10.1038/s41467-021-27901-5
[PMID: 38260422](https://pubmed.ncbi.nlm.nih.gov/38260422/)

**Parent cell type term** (check the hierarchy here https://www.ebi.ac.uk/ols4/ontologies/cl)
[intestinal epithelial cell](http://purl.obolibrary.org/obo/CL_0002563)
develops from some [intestinal crypt stem cell](http://purl.obolibrary.org/obo/CL_0002250)

**Anatomical structure where the cell type is found** (check Uberon for anatomical structures: https://www.ebi.ac.uk/ols4/ontologies/uberon)
[epithelium of crypt of Lieberkuhn](http://purl.obolibrary.org/obo/UBERON_0011184)

**Your ORCID**

**Additional notes or concerns**

Suggestion: EEC progenitor is also present in other organisms ie. flies have EEC progenitors but don't have intestinal crypts and don't express the same markers - instead they are Dl+ Pros+. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4325374/

We can have the parent class as intestinal enteroendocrine progenitor that is `epithelial cell` and part of some `intestinal epithelium ` and develops from `intestinal stem cell`. Then have a subclass that is specific to mammals?

### PR DESCRIPTION
**Pull request by Caroline-99**
Added a new term with textual and logical defs, synonyms and references

### CODE REVIEW

**Comment by dosumis** [comment]
#gogoeditdiff

**Comment by Caroline-99** [comment]
#gogoeditdiff

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -32490,6 +32491,19 @@ AnnotationAssertion(rdfs:label obo:CL_4052004 "intestinal fibroblastic reticular
 SubClassOf(obo:CL_4052004 obo:CL_0009101)
 SubClassOf(obo:CL_4052004 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001962))
 
+# Class: obo:CL_4052006 (intestinal enteroendocrine progenitor)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31756561") Annotation(oboInOwl:hasDbXref "PMID:32531023") Annotation(oboInOwl:hasDbXref "PMID:35913117") Annotation(oboInOwl:hasDbXref "PMID:38260422") Annotation(oboInOwl:hasDbXref "doi:/10.1038/s41467-021-27901-5") obo:IAO_0000115 obo:CL_4052006 "An epithelial cell that is part of the crypt of Lieberkuhn, originating from intestinal stem cells and giving rise to enteroendocrine cells (EECs). In mouse and human, it expresses markers like Neurog3, and has the ability to proliferate and differentiate into various EEC subtypes. Its proliferative potential supports both crypt growth and cell proliferation, distinguishing it from fully differentiated EECs.")
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:32496]: > In mouse and human, it expresses markers like Neurog3

This is a bit informal consider alternative formulas:
- In mouse and human, this cell expresses Neurog3
- In mouse and human, this cell can be characterized by the expression of Neurog3
- In mouse and human, this cell expresses the marker Neurog3

Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:32496]: I am super picky here, but instead of 'various' I would use 'multiple' or 'a range of'.

Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:32496]: In the sentence:
> Its proliferative potential supports both crypt growth and cell proliferation, ...

What does it mean that the proliferative potential supports cell proliferation? I am not sure if it is its own proliferation or other cells, but in any case I would remove.

**Review by aleixpuigb — CHANGES_REQUESTED** [review]
Please, see inline comments.

**Review by aleixpuigb — APPROVED** [review]

**Pull request merged**